shaders/shader.vert

```diff
@@ -11,6 +11,8 @@ layout(set = 0, binding = 0) uniform WindowUniforms {
 	vec2 offset;
 	vec2 relative_size;
 	vec2 pixel_size;
+	vec2 zoom;
+	vec2 pan;
```

Comment:
I'm wondering, should we have these as separate uniforms, or merge them into `offset` and `relative_size`? What do you think?

Reply:
I believe I did experiment with relative size. I mean if zoom was relative size, then the image was zoomed about the lower corner of the image which was ugly. Having a separate "zoom" resulted in a zoom about origin. I have to check pan vs offset though.